content/en/docs/zero-code/obi/cardinality.md

```diff
@@ -0,0 +1,471 @@
+---
+title: OBI metrics cardinality
+menuTitle: Metrics cardinality
+
+description:
+  Overview of how to calculate the cardinality of metrics produced by a default
+  OBI installation, considering the size and complexity of the instrumented
+  environment.
+weight: 24
+keywords:
+  - Beyla
+  - eBPF
+  - cardinality
+---
+
+# OBI metrics cardinality
+
+The cardinality of [OBI metrics](../metrics/) highly depends on the size and
+complexity of the instrumented environment, so there is no way to provide a
+simple and accurate formula.
+
+This document tries to provide an approximation of the metrics cardinality that
+might be produced by a default OBI installation. It is divided into several
+sections for each type of metric that OBI can produce, as each metric family
+can be selectively enabled or disabled.
+
+For simplicity, the formulas below assume a single cluster. You should multiply
+the cardinality for each of your clusters.
+
+## Terminology
+
+Before continuing, we should clarify some terms that might be vague or subject
+to interpretation:
+
+- **Instance**: is each instrumentation target. In application-level metrics, it
+  would be the service or client instance. In Kubernetes, it would be a Pod. In
+  process-level metrics, each instance is each reported process. An application
+  instance might run in multiple processes. In network-level metrics, each
+  instance is the OBI instance that instruments all the network flows in a
+  given host.
```

Comment:
```suggestion
- **Instance**: is each instrumentation target. In application-level metrics, it
  would be the service or client instance. In Kubernetes, it would be a Pod. An application
  instance might run in multiple processes. In network-level metrics, each
  instance is the OBI instance that instruments all the network flows in a
  given host.
```